pkg/investigations/investigation/investigation_test.go

```
@@ -542,6 +544,100 @@ func TestResourceBuilder_HCPFields_CanBeSet(t *testing.T) {
+// correctly stores the production OCM/backplane clients and managedCloud in the built resources.
+func TestNewResourceBuilder_StoresProductionClients(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockBpClient := mock_backplane.NewMockClient(ctrl)
+	mockBpProdClient := mock_backplane.NewMockClient(ctrl)
+
+	managedCloud := &managedcloud.Client{}
+
+	rb, err := NewResourceBuilder(
+		nil,             // ocmClient (staging)
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)